modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/video/index.js

```diff
@@ -1,31 +1,31 @@
-let handleProviderResize = function() {};
```

Comment:
@p2kmgcl: As explained in [a52850eb6ecc](https://github.com/liferay-frontend/liferay-portal/pull/424/commits/a52850eb6ecc74cda0b144c5e4b22b85877207d8), I believe this is safe, but giving you a heads up just in case.